pages/index.js

```diff
@@ -21,7 +23,7 @@ const Index = () => {
         imgUrl="https://ichef.bbci.co.uk/childrens-responsive-ichef-ck/400xn/amz/cbbc/bp-duckling.jpg"
         imgAltText="duckling"
         buttonText="Learn more about Miricyl's vision"
-        buttonLink="/about"
+        buttonLink="/mental-health"
```

Comment:
What is this prop being used for? It looks to me like this prop isn't used at all in the `Header` component

Reply:
It is used in CalloutBoxButton

Reply:
Ha! Sorry, ignore me, I didn't spot I needed to expand the view on Github